### DISCUSSION TITLE
cannot access '/usr/local/bin/*'

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#209] [Fixed Issues]
FWIW Update to **rAudio 1 • 202102211** generates at the end:
`chmod: 
cannot access '/usr/local/bin/*'
: No such file or directory`

RPI 3B + HifiBerry Digi+ Pro + rAudio-1 + aarch64 (experimental) built with rOS

**Comment by klassikmann** [comment]
Not seen at update to 20210223, assuming fixed.

**Reply by rern** [reply]
Yes, it was.
